### PR TITLE
Doc Fixes

### PR DESCRIPTION
**Pull request by nikmartin**
Fixes typo of 'Instalation' in several places
Better aligns getting started page with current release. 
Deletes registration section on Getting Started Page
This may need updated screen shots, but the existing ones are not too far off from current release.

### CODE REVIEW

File: docs/manual/getting-started/index.md [hunk 1]
```diff
@@ -1,78 +1,83 @@
...
-
-This window also offers the opportunity to **Promote my station on Sourcefabric.org** (on the page <http://www.sourcefabric.org/en/airtime/whosusing/>) by checking the box. Fill in the form which will appear with some details about your station. The contact details are only requested for verification purposes, and will not be made available to the public. Click the **Browse** button to select a **Station Logo** image from the file manager on your computer.
-
-![](static/Screenshot391-Promote_my_station.png)
 
 The Master Panel
 ----------------
 
-After the pop-up window is closed, you should now see the **Master Panel**, which is present at the top of every page of the Airtime interface. On the left hand side, the Master Panel displays the details of the **Previous** file played out, the current file playing (with an orange progress bar and time elapsed/time remaining), and the details of the **Next** file due to play. It also displays the name and scheduled time of the current show, with a blue progress bar.
+After the pop-up window is closed, you should now see the **Master Panel**, which is present at the top of every page of
```
Comment (frecuencialibre) [inline docs/manual/getting-started/index.md]: Remove the now-confusing reference to the pop-up? Maybe just "The **Master Panel** can be found at top of every screen of the LibreTime interface.

**Comment by frecuencialibre** [comment]
this looks great! @nikmartin would it be quick to submit this without the added line breaks? i've seen mkdocs get unexpectedly picky about those. Also I made one inline content.

**Comment by nikmartin** [comment]
@frecuencialibre yes, I'd be glad to remove the extra line breaks, although I did build and test with mkdocs and all the compiled html content looked fine. I'll also remove the reference to the pop-up

**Comment by nikmartin** [comment]
whoops, codenvy timed out my VM: http://node21.codenvy.io:47277/nikmartin-libretime/build/docs/

**Comment by frecuencialibre** [comment]
many thanks @nikmartin ! 

I haven't used the import script, but think that docs should be conservative. also i did see the codenvy quickly, and you were right that the line breaks had no impact. in my case i had to disable automatic markdown code formatting by the pretty extension in vs code. perhaps only useful in order to keep commits a little cleaner. awesome!

**Pull request merged**